index.html

```diff
@@ -0,0 +1,136 @@
+        <div class="container">
+          <h2 class="sub-title">Main Program</h2>
+          <ul class="program-container">
+            <li class="program">
+              <img src="images/icons/lecture.png" alt="lecture-icon">
+              <h4>Lecture</h4>
+              <p>You are going to fail in love with nature, Enjoy the flexibility of a private vehicle.</p>
+            </li>
+            <li class="program">
+              <img src="images/icons/exhibition.png" alt="exhibition-icon">
+              <h4>Exhibition</h4>
+              <p>Live adventures like you never have before, discover Egypt's pyramids and giza</p>
+            </li>
+            <li class="program">
+              <img src="images/icons/forum.png" alt="forum-icon">
+              <h4>Meet Nature</h4>
+              <p>Come explore the Pacific Coast of California with us, or join one of our multi-day tripes
+              </p>
+            </li>
+            <li class="program">
+              <img src="images/icons/workshop.png" alt="workshop-icon">
+              <h4>Find your Way</h4>
+              <p>We will help you save a heap of time and money by creating your personalized travel
+                itinerary</p>
+            </li>
+            <li class="program">
+              <img src="images/icons/ignite.png" alt="ignite-icon">
+              <h4>Explore the World</h4>
+              <p>From big wows to hidden gems, our tours leave you feeling that you've really explored.</p>
+            </li>
+          </ul>
+          <a href="#" class="about-btn">Discover Our Tours</a>
+        </div>
+      </section>
+
+      <section class="speakers-section" id="stories">
+        <div class="container">
+          <h2 class="sub-title">People's Stories</h2>
+          <ul class="speakers-container"></ul>
+          <button type="button" class="show-btn"> MORE &darr;</button>
+        </div>
+      </section>
+
+      <section class="partner-section" id="partners">
+        <div class="container">
+          <h2 class="sub-title">Partner</h2>
+          <ul class="partners-container">
+            <li class="partner"><img src="images/partners/mozilla.png" alt=""></li>
+            <li class="partner"><img src="images/partners/Google.png" alt=""></li>
+            <li class="partner"><img src="images/partners/navar.png" alt=""></li>
+            <li class="partner"><img src="images/partners/daumkakao.svg" alt=""></li>
+            <li class="partner"><img src="images/partners/airbnb.png" alt=""></li>
+          </ul>
+        </div>
+      </section>
+
+    </main>
+
+    <footer>
+      <div class="container footer">
+        <img src="images/logo.png" alt="">
+        <div>
+          <h6>&copy; 2021 Green Travel websit. some rights reserved.</h6>
+          <p>The above copyright notice and this permission notice shall be included in all copies or substantial
+            portions of the [Green-Travel website].</p>
+        </div>
+      </div>
+    </footer>
```

Comment:
- [ ] According to the original design, the partners and footer sections should not be shown in the **mobile version**, the last section for the mobile version should be the speakers (or people's stories in your project). Please, kindly address this issue in the mobile version in order to match the design :smiley: :raised_hands:

| Original design | Your design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/81522435/147131660-823d562f-6346-4262-9854-f7292853388d.png) | ![image](https://user-images.githubusercontent.com/81522435/147131701-cc052be8-6cf2-4855-8d3d-378f8f585c7b.png) |

- [ ] Also, there is a similar issue for the **desktop version** because in the about page, you are showing the partners section and according to the original design it should be the footer with black background (please, keep in mind that for the mobile version and the homepage for the desktop version the color of the footer should be white :pushpin:) :smiley:

| Original design | Your design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/81522435/147132226-e59516be-4d17-4c30-839c-ef4bd856e737.png) | ![image](https://user-images.githubusercontent.com/81522435/147132255-c40438f7-1775-410f-ade5-0ef93ed12ff9.png) |